### ISSUE TITLE
Playing videos on chrome through the program?

### ISSUE THREAD

**Issue by shanlol** [LorenzCK/OnTopReplica#101]
When i try to capture a video thats playing on chrome (youtube, netflix etc), the video though the program freezes the second i minimize the window the video is playing from. Is this "intended" or am i missing something?

**Comment by VackerSimon** [comment]
Yes, the way OnTopReplica captures the window "stops working" when you minimize it

**Comment by movaid7** [comment]
For anyone else experiencing this, try the fix here ➜ https://github.com/LorenzCK/OnTopReplica/issues/103#issuecomment-706632747
